static/src/stylesheets/module/facia-garnett/item-types/_fc-item--type-liveblog.scss

```diff
@@ -45,10 +45,8 @@
         }
 
         .fc-sublink__title {
-            font-weight: 600;
-
             &::before {
-                border-top-color: #e1e1e1;
+                border-top-color: $color2;
```

Comment:
Do we have a garnett color for this? Should try use only garnet color vars so it's easier to unpick old from new next year

Reply:
$colour2 is just passed in the mixin, and gets given one of the garnett colours.